pkg/addons/adminconsole/adminconsole.go

```diff
@@ -218,6 +217,7 @@ func (a *AdminConsole) GenerateHelmConfig(onlyDefaults bool) ([]v1beta1.Chart, [
 		if err := a.addKotsApplicationToHelmValues(); err != nil {
 			return nil, nil, fmt.Errorf("unable to add kots app to helm values: %w", err)
 		}
+		helmValues["embeddedClusterID"] = metrics.ClusterID().String()
```

Comment:
hmm, does this mean that there will be no `embeddedClusterID` when only defaults are used? TBH i'm not sure what `onlyDefaults` indicates.

Reply:
onlyDefaults is what we pass in when running the `version metadata` command - it's what generates the metadata file that is associated with a embedded cluster version

it's shared, and thus really should not have an ID